### PR TITLE
Added boxes for the IFAK and First Aid Kit.

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

SUMMARY: Content "Add containers for the IFAK and First Aid Kit."

#### Purpose of change

The containers for the first aid kit and IFAk disappear to... somewhere when opened, which doesn't make sense. This PR adds them in.

#### Describe the solution

Added in the containers for the IFAK.

### CODE REVIEW

File: data/json/items/gun/shot.json [hunk 1]
```diff
@@ -852,10 +852,10 @@
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "shot": 6 } } ]
   },
   {
-    "id": "slamfire_shotgun",
+    "id": "four_winds_shotgun",
```
Comment (Rivet-the-Zombie) [inline data/json/items/gun/shot.json:855]: Commit leakage?

Comment (ZhilkinSerg) [inline data/json/items/gun/shot.json:855]:
```suggestion
    "id": "slamfire_shotgun",
```

File: data/json/items/gun/shot.json [hunk 2]
```diff
@@ -852,10 +852,10 @@
...
     "copy-from": "shotgun_base",
     "type": "GUN",
-    "name": { "str": "slam-fire pipe shotgun" },
+    "name": { "str": "four winds shotgun" },
```
Comment (ZhilkinSerg) [inline data/json/items/gun/shot.json:858]:
```suggestion
    "name": { "str": "slam-fire pipe shotgun" },
```

File: data/json/items/migration.json [hunk 3]
```diff
@@ -1281,6 +1282,6 @@
   {
     "id": "four_winds_shotgun",
     "type": "MIGRATION",
-    "replace": "slamfire_shotgun"
+    "replace": "four_winds_shotgun"
```
Comment (ZhilkinSerg) [inline data/json/items/migration.json:1285]:
```suggestion
    "replace": "slamfire_shotgun"
```

File: data/json/items/migration.json [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (ZhilkinSerg) [inline data/json/items/migration.json:1]:
```suggestion
```

**Pull request merged**